README.md

```diff
@@ -3,13 +3,14 @@
 </p>
 
 [![Static Analysis + Unit + Integration](https://github.com/anchore/grype/workflows/Static%20Analysis%20+%20Unit%20+%20Integration/badge.svg)](https://github.com/anchore/grype/actions?query=workflow%3A%22Static+Analysis+%2B+Unit+%2B+Integration%22)
-[![Acceptance](https://github.com/anchore/grype/workflows/Acceptance/badge.svg)](https://github.com/anchore/grype/actions?query=workflow%3AAcceptance)
+![Validations](https://github.com/anchore/grype/workflows/Validations/badge.svg)
```

Comment:
Why did this change?

Reply:
looks like the acceptance test badge is broken on main 

<img width="819" alt="Screenshot 2023-09-26 at 1 28 59 PM" src="https://github.com/anchore/grype/assets/590471/a34d2f20-308f-4e4c-a262-78dc48af245a">

looks like it's fixed on this branch 🙌 
<img width="994" alt="Screenshot 2023-09-26 at 1 29 48 PM" src="https://github.com/anchore/grype/assets/590471/d28781e6-f563-4c0e-ac3e-ac705c09a471">